### PR TITLE
Humanized errors can be looked from parent schemas

### LINKED ISSUE #86: Feature request or bug: adding properties directly to map keys.
Hi Tommi!

I was surprised that it is not possible to add properties to keys directly, but I don't know if it is intended behavior or not. I found this when I was playing around with error messages, so the example focuses on that. As you can see the first expression evaluates with the error message, while the second one does not.
```clj
(-> [:map [:leet [:re {:error/message "should be '1337'"} #"^1337$"]]]
    (m/explain {:leet "1338"})
    (me/humanize {:wrap :message})) ;; => {:leet "should be '1337'"}

(-> [:map [:leet {:error/message "should be: '1337'"} #"^1337$"]]
    (m/explain {:leet "1338"})
    (me/humanize {:wrap :message})) ;; => {:leet "unknown error"}
```
If this is intended behavior, maybe the the fact that the second example does not work could be included in the docs.

### PR DESCRIPTION
**Pull request by ikitommi**
using a custom `:resolve` impl with `me/humanize`:

```clj
(-> [:map
     [:foo {:error/message "entry-failure"} :int]]
    (m/explain {:foo "1"})
    (me/humanize {:resolve me/resolve-root-error}))
; => {:foo ["entry-failure"]}
```

fixes #86

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -535,6 +535,26 @@ Errors can be targeted using `:error/path` property:
 ; {:password2 ["passwords don't match"]}
 ```
 
+By default, only direct erronous schema properties are used:
```
Comment (viesti) [inline README.md:538]:
```suggestion
By default, only direct erroneous schema properties are used:
```

File: src/malli/error.cljc [hunk 2]
```diff
@@ -267,13 +283,21 @@
                    $))))))))
 
 (defn humanize
+  "Humanized a explanation. Accepts the following optitons:
+
+  - `:wrap`, a function of `error -> message`, defaulting ot `:message`
```
Comment (viesti) [inline src/malli/error.cljc:288]:
```suggestion
  - `:wrap`, a function of `error -> message`, defaulting to `:message`
```

**Comment by ikitommi** [comment]
thanks!

File: src/malli/error.cljc [hunk 3]
```diff
@@ -134,7 +136,7 @@
 (defn- -put [x k v]
   (cond
     (set? x) (conj x v)
-    (associative? x) (update x k (fn [e] (if (-just-error? v) (into (vec e) v) v)))
+    (associative? x) (update x k (fn [e] (if (-just-error? v) (vec (distinct (into (vec e) v))) v)))
```
Comment (ikitommi) [inline src/malli/error.cljc:139]: this is slow, but thinking we'll change this to (ordered?) set later anyways.

**Pull request merged**